jumpcloud-ADMU/Powershell/ProgressForm.ps1

```diff
@@ -0,0 +1,193 @@
+    [System.Reflection.Assembly]::LoadWithPartialName('presentationframework') | Out-Null
+    $syncHash = [hashtable]::Synchronized(@{ })
+    $newRunspace = [runspacefactory]::CreateRunspace()
+    $syncHash.Runspace = $newRunspace
+    $syncHash.PercentComplete = $PercentComplete
+    $syncHash.StatusInput = ''
+    $syncHash.LogText = @()
+    $synchash.logLevel = ''
+    $syncHash.base64JCLogo = DecodeBase64Image -ImageBase64 $newJCLogoBase64
+    $synchash.closeWindow = $false
+    $synchash.scriptPath = $scriptPath
+
+    $syncHash.XAML = @"
+    <Window
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    Name="Window" Title="JumpCloud ADMU Migration..."
+    WindowStyle="SingleBorderWindow"
+    ResizeMode="NoResize"
+    Background="White" ScrollViewer.VerticalScrollBarVisibility="Visible" ScrollViewer.HorizontalScrollBarVisibility="Visible" Width="626" Height="539">
+    <Grid Margin="0,0,0,3">
+        <TextBlock Name="Status" FontWeight="Medium" FontSize="14" TextAlignment="Center" HorizontalAlignment="Center" TextWrapping="Wrap"  VerticalAlignment="Center" Width="422" Height="70"/>
+        <ProgressBar Name="ProgressBar" HorizontalAlignment="Center" Height="37" Margin="0,147,0,0" VerticalAlignment="Top" Width="422" Foreground="#FF104357"/>
+        <Button Name="ViewLog" Content="View Log" HorizontalAlignment="Left" Margin="52,440,0,0" VerticalAlignment="Top" Height="27" Width="69" Visibility="Hidden" />
+        <Button Name="StartJCADMU" Content="RunJCAdmu" HorizontalAlignment="Left" Margin="505,441,0,0" VerticalAlignment="Top" Height="26" Width="69" Visibility="Hidden" />
+        <Expander HorizontalAlignment="Center" Height="127" Header="Log" Margin="0,297,0,0" VerticalAlignment="Top" Width="522">
+            <Expander.Content>
+                <ScrollViewer Name="ScrollLog" Margin="0,10,0,9" Foreground="Gray" HorizontalScrollBarVisibility = "Auto">
+                    <TextBlock Name="LogTextBlock" TextWrapping="Wrap" FontWeight="Medium" FontSize="12" >
+                    </TextBlock>
+                </ScrollViewer>
+            </Expander.Content>
+        </Expander>
+        <Image Name="JCLogoImg" Margin="102,41,102,362" Stretch="Fill"/>
+    </Grid>
+</Window>
+"@
+
+    # Create a runspace to run the form in
+    $newRunspace.ApartmentState = "STA"
+    $newRunspace.ThreadOptions = "ReuseThread"
+    $data = $newRunspace.Open() | Out-Null
+    $newRunspace.SessionStateProxy.SetVariable("syncHash", $syncHash)
+
+    # Add the form code to the powershell instance
+    $psCommand = [PowerShell]::Create().AddScript({
+        # Load an xaml form
+
+        $syncHash.Window = [Windows.Markup.XamlReader]::parse( $SyncHash.XAML )
+        ([xml]$SyncHash.XAML).SelectNodes("//*[@Name]") | % { $SyncHash."$($_.Name)" = $SyncHash.Window.FindName($_.Name) }
+        # Image
+        $SyncHash.JCLogoImg.Source = $syncHash.base64JCLogo
```

Comment:
I've seen at least a few iterations of running the script where this fails to load. We can probably cut down on the size of source image you've used here to improve performance.

![Screenshot 2024-05-23 at 12 32 46 PM](https://github.com/TheJumpCloud/jumpcloud-ADMU/assets/54448601/bb51c6bb-bb22-4c18-9ffd-e8c6f043338c)